setup.py

```diff
@@ -89,6 +110,14 @@ def findRequirements():
   requirementsPath = os.path.join(REPO_DIR, "requirements.txt")
   requirements = parse_file(requirementsPath)
 
+  if nupicBindingsPrereleaseInstalled():
+    # User has a pre-release version of nupic.bindings installed, which is only
+    # possible if the user installed and built nupic.bindings from source and
+    # it is up to the user to decide when to update nupic.bindings.  We'll
+    # quietly remove the entry in requirements.txt so as to not conflate the
```

Comment:
This comment sounds a bit like the code is going to modify the `requirements.txt` file. "quietly remove the entry **from requirements** so as..." might be less confusing.